pkg/log/messages/error_messages.go

```diff
@@ -20,7 +20,7 @@ const CSPFK003E string = "CSPFK003E AccessToken failed to delete access token da
 // Environment variables
 const CSPFK004E string = "CSPFK004E Environment variable '%s' must be provided"
 const CSPFK005E string = "CSPFK005E Provided incorrect value for environment variable %s"
-const CSPFK007E string = "CSPFK007E Setting SECRETS_DESTINATION environment variable to 'k8s_secrets' must run as init container"
+const CSPFK007E string = "CSPFK007E CONTAINER_MODE '%s' is not supported. Supported values are: %v"
```

Comment:
can you check if this log code is written in the docs?

Reply:
<img width="973" alt="Screen Shot 2020-08-13 at 11 01 47 AM" src="https://user-images.githubusercontent.com/19418506/90114277-2d4ced00-dd5b-11ea-8a87-ead9d15a6899.png">

it is

Reply:
<img width="589" alt="Screen Shot 2020-08-13 at 11 49 33 AM" src="https://user-images.githubusercontent.com/19418506/90114299-363dbe80-dd5b-11ea-8a64-eb68c44b4f8a.png">
it is

Reply:
so we need to update that. can you file an issue that changes the issue and resolution?

Reply:
Here we are: https://github.com/cyberark/conjur-docs/issues/1079